gef.py

```diff
@@ -2814,19 +2817,19 @@ def instantiate(self, base):
             err(e)
             return
 
-        if 'Breakpoint' not in res:
+        if "Breakpoint" not in res:
             err(res)
             return
-        res_list = res.split(' ')
+        res_list = res.split(" ")
```

Comment:
Quick Python tip: prefer `<string>.split()` instead of `<string>.split(" ")`.

```python
>>> "   a  a a   a".split()
['a', 'a', 'a', 'a']                         # which is usually what we want
>>> "   a  a a   a".split(" ")
['', '', '', 'a', '', 'a', 'a', '', '', 'a'] # will produce heaps of empty elements
```

Reply:
It'll be fix shortly.